app/controllers/users_controller.rb

```diff
@@ -1,24 +1,42 @@
 class UsersController < ApplicationController
+  before_action :set_user, only: [:show,:edit_password, :update, :update_pwd,:destroy]
```

Comment:
refactor [:show,:edit_password, :update, :update_pwd,:destroy] to
`%i[show edit_password update update_pwd destroy]`